### DISCUSSION TITLE
Xcode project on external HD

### DISCUSSION THREAD

**Discussion by phr85** [pointfreeco/swift-snapshot-testing#911] [General]
Dear community

As the build server (macmini) only has 265GB of internal space, I had to move the project to an external SSD. 
The problem now is that when I run snapshot tests, I get permission denied for snapshots stored on external HD. I have given 777 permissions for all files on the external drive. I also try many things but always end up with permission denied.

Before I was running it from internal drive which was no problem and I didn't have any permission issue.

Does anyone have any idea what else I could try?

Best 

Phil

![2024-09-20_11-20](https://github.com/user-attachments/assets/bce07ab5-7132-4d76-9450-a264266c62c4)

**Comment by mbrandonw** [comment]
Hi @phr85, I don’t think this is an issue with the library so I am going to convert it to a discussion. Hopefully someone can help you over there!
